engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -489,10 +491,13 @@ private void updateCamera(CharacterMovementComponent charMovementComp, Vector3f
             }
         }
 
-        if (charMovementComp.mode == MovementMode.GHOSTING) {
-            playerCamera.extendFov(24);
-        } else {
-            playerCamera.resetFov();
+        if(charMovementComp.mode != lastMode){
+            if (charMovementComp.mode == MovementMode.GHOSTING) {
```

Comment:
> I was also thinking that terms such as spectator, observer and watcher might have their use in specific game modes.

Yep, agreed.

`incorporeal` is a cool term, even if a bit long. `wisp` is short and kinda neat, but pretty close to `ghost` and not obvious. Maybe we should drop in a TODO or issue to consider the rename as part of the movement refactor since that could involve tweaking at the movement mode. Maybe of interest to @portokaliu